frontend/src/metabase-lib/Question.ts

```diff
@@ -1221,6 +1221,12 @@ class Question {
       );
     }
 
+    // Helpers for working with the current query from CLJS REPLs.
+    if (process.env.NODE_ENV === "development") {
+      window.__MLv2_metadata = metadata;
```

Comment:
How do you access this in the REPL?

Reply:
With `(dev.fe-helpers/current-query)`. See https://github.com/metabase/metabase/pull/34997/files#diff-5e5e26a1cd736407870cb6c806d437ee1698e39a4d6f662c32db2f496c3a3c0dR14 which is accessing this hack on `window`.